trlx/trainer/accelerate_base_trainer.py

```diff
@@ -281,6 +281,8 @@ def learn(self):
 
                     stats["time/forward"] = forward_time
                     stats["time/backward"] = backward_time
+                    for group_number, lr in enumerate(self.scheduler.get_last_lr()):
+                        stats[f"learning_rate/group_{group_number}"] = lr
```

Comment:
Currently, we're logging a ton of metrics: 
<img width="190" alt="image" src="https://user-images.githubusercontent.com/41410219/209216860-268cc39e-3b4a-4308-9805-e2e12ca33860.png">
so to avoid adding extra clutter, I think it might be best to keep the LR dynamics tracked in `Charts` by using the following stats key instead: `stats[f"learning_rate_group_{group_number}"] = lr`

Reply:
Agree. Moved it to charts

<img width="552" alt="image" src="https://user-images.githubusercontent.com/25286019/209311037-8854a60a-d037-4fd9-90a4-4da7ee6af688.png">